app/utils/android-helpers.ts

```diff
@@ -0,0 +1,175 @@
+import messaging, { FirebaseMessagingTypes } from "@react-native-firebase/messaging"
+import * as Burnt from "burnt"
+import notifee, { AndroidImportance, AndroidLaunchActivityFlag } from "@notifee/react-native"
+import { RideState, RideStatus, getStatusEndDate, rideProgress } from "../hooks/use-ride-progress"
+import { RideApi, RouteItem } from "../services/api"
+import { findClosestStationInRoute, getRideStatus, getTrainFromStationId } from "./helpers/ride-helpers"
+import { addMinutes, differenceInMinutes, format } from "date-fns"
+import Preferences from "react-native-default-preference"
+import { getInitialLanguage, translate } from "../i18n"
+import i18n from "i18n-js"
+
+const rideApi = new RideApi()
+let unsubscribeTokenUpdates: () => void
+
+const getRideNotificationId = () => Preferences.get("rideNotificationId")
+const setRideRoute = (route: RouteItem) => Preferences.set("rideRoute", JSON.stringify(route))
+const setRideNotificationId = (notificationId: string) => Preferences.set("rideNotificationId", notificationId)
+const getRideRoute = async () => {
+  const savedRoute = await Preferences.get("rideRoute")
+  return savedRoute && (JSON.parse(savedRoute) as RouteItem)
+}
+
+export const configureAndroidNotifications = async () => {
+  notifee.createChannel({
+    id: "better-rail",
+    name: "Better Rail",
+    description: "Get live ride notifications",
+    importance: AndroidImportance.HIGH,
+    sound: "default",
+  })
+
+  notifee.createChannel({
+    id: "better-rail-live",
+    name: "Better Rail Live",
+    description: "Get live ride persistent notification",
+    vibration: false,
+  })
+
+  const onRecievedMessage = async (message: FirebaseMessagingTypes.RemoteMessage) => {
+    if (message.notification) {
+      Burnt.toast({
```

Comment:
Will the toast will be displayed in the live ride screen?

Reply:
Yes, it will be displayed when the app is in foreground, it'll show the user instruction that the current bottom sheet doesn't show.

Reply:
Yes, it will be displayed when the app is in foreground, it'll show the user instruction that the current bottom sheet doesn't show.